### ISSUE TITLE
Stunned animals never lose the stunned tag

### ISSUE THREAD

**Issue by CharlieTheQ** [zebediah49/PetProtection#2]
Tamed animals stunned by enemy attacks keep the stunned tag forever, even if they've been fed, are at full health, etc.  Turning the mod off removes the stunned tag, but turning it back on replaces it.

**Comment by HenrikPoulsen** [comment]
I agree with this. I have had a Lox that has been stunned for many many hours. It never goes away

**Comment by GrarUlf** [comment]
Same problem, it's a shame that this is the only mod that makes animals unkillable, yet the author of the mod is probably not going to update it... I lost 2 of my two-star wolves because they were stunned somewhere in the woods behind me, and I never found them again. So dumb.
